### ISSUE TITLE
cracklib-format thinks that cracklib-words is a binary file

### ISSUE THREAD

**Issue by danomimanchego123** [cracklib/cracklib#14]
Hi,

On Lubuntu 16.04, the `grep` in `cracklib-format` thinks that the  `cracklib-words` is a binary file:

```
$ . src/util/cracklib-format words/cracklib-words.gz 

]
]]
binaryfilestandardinputmatches
```
However, this problem can be avoided if you add a `-a` to the `grep` in `cracklib-format`, to force `grep` to treat the input as text:

```diff
$ git diff
diff --git a/src/util/cracklib-format b/src/util/cracklib-format
index 1d7be5b..b1de8e8 100644
--- a/src/util/cracklib-format
+++ b/src/util/cracklib-format
@@ -4,7 +4,7 @@
 # into cracklib-packer
 #
 gzip -cdf "$@" |
-    grep -v '^\(#\|$\)' |
+    grep -a -v '^\(#\|$\)' |
     tr '[A-Z]' '[a-z]' |
     tr -cd '\012[a-z][0-9]' |
     env LC_ALL=C sort -u

```

**Comment by tomato42** [comment]
that would indicate that the words file is malformed. `grep` handles multi-byte encoding just fine

**Comment by ffontaine** [comment]
Fixed by a1379d0081e14958b2ada6dcc1fcb7f128e4bd94, I think this issue can be closed

**Comment by vapier** [comment]
done!
